modules/multiaccount-setup/variables.tf

```diff
@@ -6,10 +6,22 @@ variable "guarduty_enabled" {
 
 variable "guarduty_s3_protection_enabled" {
   type        = bool
-  default     = true
+  default     = true // false
   description = "Whether to enable GuardDuty S3 protection or not."
 }
 
+variable "guarduty_kubernetes_protection_enabled" {
+  type        = bool
+  default     = true // false
+  description = "Whether to enable Kubernetes protection or not."
+}
+
+variable "guarduty_malware_protection_enabled" {
+  type        = bool
+  default     = true // false
```

Comment:
@marianod92 I think we should remove these commented values if they are not really needed.

Reply:
@diego-ojeda-binbash 
*Done!* 
I removed the comments from the _auto_enabled variables and forgot those. Thanks!